suite-native/module-device-onboarding/src/components/wallet-backup/CardFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { LinearGradient } from 'expo-linear-gradient';
+
+import { Box, Button, DEFAULT_INSET_BOTTOM } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { getScreenWidth } from '@trezor/env-utils';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { hexToRgba } from '@trezor/utils';
+
+import { WalletBackupType } from '../../../hooks/useWalletBackupPicker';
+
+const SCREEN_WIDTH = getScreenWidth();
+
+const containerStyle = prepareNativeStyle<{ insetBottom: number }>((_, { insetBottom }) => ({
+    position: 'absolute',
+    bottom: insetBottom,
+    width: '100%',
+}));
+
+const buttonContainerStyle = prepareNativeStyle(utils => ({
+    width: '100%',
+    backgroundColor: utils.colors.backgroundSurfaceElevation0,
+    alignItems: 'center',
+}));
+
+const buttonStyle = prepareNativeStyle(utils => ({
+    width: SCREEN_WIDTH - 2 * utils.spacings.sp16,
+}));
+
+const linearGradientStyle = prepareNativeStyle(utils => ({
+    width: '100%',
+    height: utils.spacings.sp32,
+}));
+
+interface CardFooterProps {
+    onSubmit: () => void;
+    selectedType: WalletBackupType;
+}
+
+export const CardFooter = ({ onSubmit, selectedType }: CardFooterProps) => {
```

Comment:
I would name this component differently. Like this it feels like it defines a UI of a footer that will be placed in each of the cards. But actually it is a footer of the whole bottom sheet. What about:

```suggestion
export const WalletBackupSheetFooter = ({ onSubmit, selectedType }: WalletBackupSheetFooterProps) => {
```

Reply:
Yes this is convention I am aware of, wanted to actually propose a change to this "full naming" approach, as I think if the component is used solely within one feature/component/screen its fine to have less descriptive names as we do not need to come up with super long names, but its also fine if we change it. I noticed that in some cases there are a few exceptions as well, so wanted to give it a shot :)

Reply:
thats why its in the `wallet-backup` group folder

Reply:
IMHO the descriptive naming convention is good fit for Layerd Architecture Style, but as most of the app follows Domain Driven one (folder by module/feature) we can go even with duplicated names as we know it should not be used outside of the domain

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/1dc2821ac350168d47b53527a36c4c558ebf1543

Reply:
folder structure changed https://github.com/trezor/trezor-suite/pull/17981/commits/d08e464d44e695a02cac6740d3abe5bd744cc0c7 (I left the WalletBackupCard name as it is already using a component called Card)

Reply:
There is a new folder `wallet-backup` in the components. It has a sibling `WalletBackupTutorial` that should be logically a child folder. Please move it there. Or remove the `wallet-backup` folder completely. IMO the components folder is not so complicated yet so it would be needed.

Reply:
IMO its super complicated :D Let's do a compromise and rename the `wallet-backup` to `WalletBackupSheet` what do you think?

Reply:
ok, that works